### ISSUE TITLE
According to WCAG, the smiley in the alt-text of the jetpack (hidden) image) throws a level A-error.

### ISSUE THREAD

**Issue by egullhaug** [Automattic/jetpack#18188]
Maybe not a bug-bug, but bug in the sense that WCAG-standard doesn't like the smiley in the alt-text om the img-tag that is insorted from the jetpack-plugin, and produces an error. 

To find the img-tag in question: 

1. Choose "inspect code" in whichever browser you are using. 
2. right above the </body>-tag, you'll find: 
<img src="https://pixel.wp.com/g.gif?v=ext&amp;j=1%3A9.2.1&amp;blog=125265915&amp;post=270&amp;tz=1&amp;srv=forenkling.brreg.no&amp;host=forenkling.brreg.no&amp;ref=&amp;fcp=0&amp;rand=0.25167434565778457" **alt=":)"** width="6" height="5" id="wpstats">
3. according to my sortSite-report, this is not wanted characters in an alt-field. [WCAG 2.1 A F72](https://www.w3.org/TR/WCAG20-TECHS/F72.html). 

Would you consider changing up something here so the A-level error could be avoided?  

With regards, 
Ellen

**Comment by egullhaug** [comment]
Thanks, jeherve - but how can I do that when the code comes from wp.com?

**Comment by jeherve** [comment]
@egullhaug That's a bug we'll need to fix on our end indeed.

**Comment by BrookeDot** [comment]
Hi @jeherve as this has been open for a while, I'm wondering if you can connect me with someone internally who can help make a recommendation for what the alt text should be to see if we can get this fixed.

**Comment by jeherve** [comment]
We had removed the smiley back in 2021, in D70373-code, closing #2856. This matches recommendations:

> Images that do not convey content, are decorative, or contain content that is already conveyed in text are given empty alternative text (`alt=""`) or implemented as CSS backgrounds. All linked images have descriptive alternative text.
-- https://webaim.org/standards/wcag/checklist#sc1.1.1

With that in mind, I think we can close this issue as well.
